### PR TITLE
Linux support added

### PR DESCRIPTION
**Pull request by Eric898989**
I needed to use this lib either on windows or mono linux. Working like a charm on Windows and Raspian stretch (Raspberry pi).
Eric

### CODE REVIEW

File: CoreHtmlToImage/HtmlConverter.cs [hunk 1]
```diff
@@ -10,27 +10,63 @@ namespace CoreHtmlToImage
...
+        private static string toolFilepath;
 
         static HtmlConverter()
         {
             directory = AppContext.BaseDirectory;
-            toolFilepath = Path.Combine(directory, toolFilename);
 
-            if (!File.Exists(toolFilepath))
+            //Check on what platform we are
+            if (System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform(System.Runtime.InteropServices.OSPlatform.Windows))
```
Comment (andrei-m-code) [inline CoreHtmlToImage/HtmlConverter.cs]: Do you mind changing if/else-if/else to switch/case and have a private method for each operating system.

File: CoreHtmlToImage/HtmlConverter.cs [hunk 2]
```diff
@@ -10,27 +10,63 @@ namespace CoreHtmlToImage
...
+                }
+                else
+                {
+                    throw new Exception("wkhtmltoimage does not appear to be installed on this linux system according to which command; go to https://wkhtmltopdf.org/downloads.html");
                 }
             }
+            else
+            {
+                //OSX not implemented
+                throw new Exception("OSX Platform not implemented yet");
```
Comment (andrei-m-code) [inline CoreHtmlToImage/HtmlConverter.cs]: Please throw NotSupportedException in switch default block, also don't assume it's OSX, who knows, maybe dotnet will run on Chrome OS soon :).

**Comment by BramEsendam** [comment]
Is this still going to be added?

File: CoreHtmlToImage/HtmlConverter.cs (hunk 2 above)
Comment (barvad) [reply CoreHtmlToImage/HtmlConverter.cs]: > Please throw NotSupportedException in switch default block, also don't assume it's OSX, who knows, maybe dotnet will run on Chrome OS soon :).
Are you planning to add linux support?

**Pull request merged**